src/docs/guides/decoupled-cms/README.md

```diff
@@ -81,18 +81,27 @@ builder.Services.AddOrchardCms();
 ```
 
 !!! warning "Razor Pages"
-    `AddRazorPages` must not be called directly as `services.AddOrchardCms()` already invokes it internally.
+    `builder.Services.AddRazorPages()` must not be called directly as `builder.Services.AddOrchardCms()` already invokes it internally.
 
 - Edit the `Program.cs` file
-- Remove everything after `app.UseStaticFiles();` and replace it by `app.UseOrchardCore();` like this:
+- Add `app.UseOrchardCore();` 
+- If any of the following lines exists in your `Program.cs` file, remove them:
 
 ```csharp
-   ...
-   
-   app.UseHttpsRedirection();
-   app.UseStaticFiles();
```

Comment:
It still may need this, right?
`app.UseStaticFiles();`

Reply:
@agriffard 
Yes `app.UseStaticFiles()` is still used. I don't know this isn't part of the **Static File Provider Feature**.